tools/eve-installer-gen.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+# shellcheck disable=SC2086
+# shellcheck disable=SC2181
+
+USAGE="Usage: $0 [options]
+Options:
+  -u <controller URL>
```

Comment:
This is not a URL but just the name of the server. Can't handle https:// prefix,

Reply:
Right, will change it to "-d <controller's fully-qualified **d**omain name>" ("-c" is taken for onboarding **c**ertificate)

Reply:
Changed to `-c <controller's fully-qualified domain name>`